hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -345,8 +345,10 @@ private function up1402_removedUnusedManagedEntities($caseTypes, $activityTypes)
       $entitiesToRemove[$caseType] = 'CaseType';
     }
 
-    foreach ($activityTypes as $activityType) {
-      $entitiesToRemove['civitask:act:' . $activityType] = 'OptionValue';
+    foreach ($activityTypes as $extension) {
```

Comment:
I think this is probably my fault from changing the structure of the default activity types.